src/helpers/xml-builder.js

```diff
@@ -590,6 +590,17 @@ const buildIndentation = (left = 720) => {
   return indentationFragment;
 };
 
+const buildHeading = (heading = 'Heading1') => {
```

Comment:
please find the actual default value from OOXML pdf and set that as the default value, so we can reuse this method for other instances.

Reply:
Sorry, but I have no idea what the OOXML is. I only reverse engineered a working docx-file to get this to work. Do you have an URL where I can learn more?

Will fix the other change requests within the next couple of days

Reply:
thats totally fine, this package is based on something called OOXML standard or ECMA 376 based on which docx softwares work. You can find it [here](https://www.ecma-international.org/wp-content/uploads/ecma-376_first_edition_december_2006.zip). Thanks so much for taking the effort.